javascripts/main.js

```diff
@@ -0,0 +1,33 @@
+let textInput = document.getElementById('text-input');
+
+window.addEventListener('load', function(){
+	CarLot.Inventory.loadInventory();
+});
+
+window.addEventListener('click', function() {
+	// console.log(event);
+	if (event.target.className === 'car col-sm-4') {
+		let targetCard = event.target;
+		CarLot.Inventory.clearSelectedClass();
+		targetCard.classList.toggle('selected');
+		textInput.value = '';
+		textInput.focus();
+	} else if (event.target.parentElement.className === 'car col-sm-4') {
+		let targetCard = event.target.parentElement;
+		CarLot.Inventory.clearSelectedClass();
+		targetCard.classList.toggle('selected');
+		textInput.value = '';
+		textInput.focus();
+	} else {
+		console.log("not a card");
```

Comment:
It's a good habit to console log certain things out during development, but once a project reaches production, you'll want to trim console logs out, as well as comments that are not absolutely necessary for the code.